compiler/src/model/json-spec.ts

```diff
@@ -74,5 +74,6 @@ export default function buildJsonSpec (): Map<string, JsonSpec> {
     map.set(name, json[name])
   }
 
-  return map
+  // Ensure deterministic ordering
+  return new Map([...map.entries()].sort())
```

Comment:
Maps in JS are iterated in the order of insertion. This map is created by enumerating the filesystem which causes the entries to be inserted in a different order on Linux and Windows.